### PR TITLE
Fix issue with totp password not being pipeable

### PR DESCRIPTION
**Pull request by Zaph-x**
This pull request fixes an issue with piping from stdout to 1pass.
Previously when piping to 1pass for a totp password, you would get an error saying `<item>` is empty.
This is due to `-` never being substituted for the item you are querying for.

With these changes it is now possible to call `echo "GitHub" | 1pass - totp` and get the one-time password from 1pass.

### CODE REVIEW

**Comment by dcreemer** [comment]
Thanks for this PR! I'm reviewing it now.

File: 1pass [hunk 1]
```diff
@@ -198,7 +198,6 @@ signin()
     se=$("$GPG" -d -q "$secret")
     if [ "${use_totp}" == "1" ]; then
         local ot
-        local totp
```
Comment (dcreemer) [inline 1pass:201]: Why is this removed?

**Comment by dcreemer** [comment]
This looks fine -- just one question above, then I'm happy to merge it. Thanks again!

File: 1pass (hunk 1 above)
Comment (Zaph-x) [reply 1pass:201]: The local variable has been removed to access the location of the totp file. By having it local, bash is unable to find the path to the totp file.

Sorry for the late reply. Exams happened

Comment (dcreemer) [reply 1pass:201]: Awesome -- thanks for the response!

**Comment by dcreemer** [comment]
Merged. @Zaph-x can you please test the master banch?

**Comment by Zaph-x** [comment]
@dcreemer Works as intended, thank you! 
image
